build.gradle

```diff
@@ -86,4 +86,152 @@ task loadReferenceDataStaging(type: com.marklogic.gradle.task.MlcpTask) {
   ]
 }
 
-mlLoadData.dependsOn loadReferenceDataStaging
\ No newline at end of file
+task loadMetaDataStaging(type: com.marklogic.gradle.task.MlcpTask) {
+  doFirst {
+    database = mlStagingDbName
+    port = mlStagingPort.toInteger()
+  }
+  classpath = configurations.mlcp
+  command = "IMPORT"
+  args = [
+      "-input_file_path", "$project.projectDir/src/main/ml-data/meta",
```

Comment:
Let's split out STAGING and FINAL metadata into different directories. If not now, we will eventually need to control them separately, I think.

Reply:
The problem I ran into is that it's the same reference data but it needs to be in both STAGING and FINAL because while most of the hub flows run against STAGING the patient merge step runs against FINAL. I couldn't find a way to clean this up any more than it is here in the documentation.